app.js

```diff
@@ -27,7 +48,7 @@ app.post('/trucks', function (request, response) {
 
 app.delete('/trucks/:name', function (request, response) {
 	trucks.removeTruck(request.params.name);
-
+	trucks.removeTruck(truckName);
```

Comment:
truckName is undefined - you would need to either assign request.params.name to truckName or you would need to pass request.params.name to removeTruck()